src/_posts/databases/postgresql/2000-01-01-troubleshooting.md

```diff
@@ -157,6 +157,10 @@ free to adjust this depending on your usage.
    ORDER BY cache_hit_ratio DESC
    LIMIT 10;
    ```
+   
+### Logging database statistics
+
+Database default's log drain will not contain query statistics, which are ephemeral. Plus, you have a limited time to troubleshoot runnings queries, which may not be convenient. To log them and other statistics, such as query statistics, slow queries, database I/O and CPU, you can use our [API](https://developers.scalingo.com/databases/metrics) and log them by yourself using [a scheduler](https://doc.scalingo.com/platform/app/task-scheduling/scalingo-scheduler). An open-source example is [available here](https://doc.scalingo.com/platform/app/task-scheduling/scalingo-scheduler), although not provided by Scalingo itself.
```

Comment:
@GradedJestRisk did you intend for that last link to go to https://github.com/1024pix/pix-db-stats ?

Reply:
Sure !